docs/sources/features/explore/index.md

```diff
@@ -92,9 +132,9 @@ You can customize how logs are displayed and select which columns are shown.
 
 Log data can be very repetitive and Explore can help by hiding duplicate log lines. There are a few different deduplication algorithms that you can use:
 
-* `exact` Exact matches are done on the whole line, except for date fields.
-* `numbers` Matches on the line after stripping out numbers (durations, IP addresses etc.).
-* `signature` The most aggressive deduping - strips all letters and numbers, and matches on the remaining whitespace and punctuation.
+- `exact` Exact matches are done on the whole line, except for date fields.
```

Comment:
Another example of reference information that could be put in a table.

Reply:
Why are you removing the asterisks and replacing them with hyphens? Per the [Markdown style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-markdown-guide.md), preferred bullet style is asterisk.

Reply:
> Why are you removing the asterisks and replacing them with hyphens? Per the Markdown style guide, preferred bullet style is asterisk.

It is not me, it is Prettier as it is their default setting yo use hyphens instead of asterisks.